src/components/FlexGrid/FlexGrid.js

```diff
@@ -12,18 +12,18 @@ const propTypes = {
 
 /**
  * FlexGrid is a flexbox based React component for doing layout. Under the hood it uses
- * [flexboxgrid2](https://evgenyrodionov.github.io/flexboxgrid2) and pulls in that CSS
- * using CSS Modules.
+ * a customized version of [flexboxgrid2](https://evgenyrodionov.github.io/flexboxgrid2)
+ * that removes the magic gutters used by the original flexboxgrid2 css library. This
+ * customized version (at `./flexboxgrid2-customized.scss`) is pulled in using CSS Modules.
  *
  * @public
  *
- * @param {boolean} fluid - whether to use grid in fluid mode or not
- * @param {string} className - in addition to sizing attributes, you can add custom CSS
+ * @param {string} className - in addition to sizing attributes, you can add custom CSS for the grid container
  * @param {string} tagName - what type of html tag to use for the container defaults to `<div>`
  * @param {ReactNode} children - children to render within the grid
  */
 export const FlexGrid = (props) => {
-  const containerClass = getClass(props.fluid ? 'container-fluid' : 'container')
+  const containerClass = getClass('flexgrid-container')
```

Comment:
Since this is essentially a global css class, it was conflicting with and EDS styleguide class: `.container` and so I elected to prefix it here